### PR TITLE
Add resolution to create-transcoding-job script

### PR DESCRIPTION
**Pull request by fflorent**
Add `-r [resolution]` to the `create-transcoding-job` script ;)

### CODE REVIEW

**Comment by Chocobozzz** [comment]
Please fix conflicts and now you can use `waitJob` function https://github.com/Chocobozzz/PeerTube/commit/3cd0734fd9b0ff21aaef02317a874e8f1c06e027#diff-704c6074a15946409b54e1a9ead4feffR59

**Comment by fflorent** [comment]
Yay, much faster! \o/

That's done now!

File: server/tests/cli/create-transcoding-job.ts [hunk 1]
```diff
@@ -100,6 +102,34 @@ describe('Test create transcoding jobs', function () {
...
+    for (const server of servers) {
+      const res = await getVideosList(server.url)
+      const videos = res.body.data
+      expect(videos).to.have.lengthOf(2)
+
+      let infoHashes: { [ id: number ]: string }
+
+      const video = videos.find(v => v.uuid === video1UUID)
+
+      const res2 = await getVideo(server.url, video.uuid)
```
Comment (Chocobozzz) [inline server/tests/cli/create-transcoding-job.ts]: const res2 = await getVideo(server.url, video1UUID)

File: server/tests/cli/create-transcoding-job.ts [hunk 2]
```diff
@@ -100,6 +102,34 @@ describe('Test create transcoding jobs', function () {
...
+    await execCLI(`${env} npm run create-transcoding-job -- -v ${video1UUID} -r 480`)
+
+    await waitJobs(servers)
+
+    for (const server of servers) {
+      const res = await getVideosList(server.url)
+      const videos = res.body.data
+      expect(videos).to.have.lengthOf(2)
+
+      let infoHashes: { [ id: number ]: string }
```
Comment (Chocobozzz) [inline server/tests/cli/create-transcoding-job.ts]: You don't use it

**Comment by fflorent** [comment]
I have taken into account your remarks. Tell me when all is good to you so I can fixup the commits ;).

Florent

File: server/tests/cli/create-transcoding-job.ts [hunk 3]
```diff
@@ -100,6 +102,32 @@ describe('Test create transcoding jobs', function () {
...
+    await execCLI(`${env} npm run create-transcoding-job -- -v ${video1UUID} -r 480`)
+
+    await waitJobs(servers)
+
+    for (const server of servers) {
+      const res = await getVideosList(server.url)
+      const videos = res.body.data
+      expect(videos).to.have.lengthOf(2)
+
+      const video = videos.find(v => v.uuid === video1UUID)
```
Comment (Chocobozzz) [inline server/tests/cli/create-transcoding-job.ts]: What's the point of this?

Comment (fflorent) [reply server/tests/cli/create-transcoding-job.ts]: Powerful rhetorical question. This line is removed :).

**Comment by Chocobozzz** [comment]
Thanks @fflorent

**Pull request merged**